react/react1/week1/todo_list/src/components/ListItem.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./ListItem.css";
+
+const ListItem = ({ todo, time, id }) => {
+  return (
+    <div>
+      <li>
+        <h3>
```

Comment:
You don't need to add h3 inside. Simple li should work here. You don't need to show the {id} as it is not described in the exercise. It should look like the image below :)

![image](https://github.com/arash8877/hyf-homework/assets/1160546/616b7560-9560-47cf-8ee6-98d2b64f63a7)